apps/emqx_plugins/src/emqx_plugins.erl

```diff
@@ -665,6 +674,7 @@ do_load_plugin_app(AppName, Ebin) ->
     lists:foreach(
         fun(BeamFile) ->
             Module = list_to_atom(filename:basename(BeamFile, ".beam")),
+            _ = code:purge(Module),
```

Comment:
Plugins failed to get started with a `not_purged` error if the user changes his config and then try to start it again.